### PR TITLE
Update query.md

### PR DESCRIPTION
**Pull request by arthuo**
Overwrite with new ones_zh

### CODE REVIEW

File: site/zh-CN/userGuide/search/query.md [hunk 1]
```diff
@@ -4,20 +4,20 @@ related_key: query vectors
...
 
-{{fragments/translation_needed.md}}
 
-This topic describes how to conduct a vector query.
 
-Unlike a vector similarity search, a vector query retrieves vectors via scalar filtering based on [boolean expression](boolean.md). Milvus supports many data types in the scalar fields and a variety of boolean expressions. The boolean expression filters on scalar fields or the primary key field, and it retrieves all results that match the filters.
+当前主题介绍如何进行向量结构化匹配。
 
-The following example shows how to perform a vector query on a 2000-row dataset of book ID (primary key), word count (scalar field), and book introduction (vector field), simulating the situation where you query for certain books based on their IDs.
+与向量相似性搜索不同，向量结构化匹配通过基于[布尔表达式](boolean.md) 的标量过滤来检索向量。 Milvus 支持标量 field 中的多种数据类型和多种布尔表达式。布尔表达式过滤标量 field 或者主键 field，并检索与过滤器匹配的所有结果。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/query.md:13]: 主键 field --》primary key field

File: site/zh-CN/userGuide/search/query.md [hunk 2]
```diff
@@ -55,11 +55,11 @@ milvusClient.loadCollection(
...
 ```
 
-## Conduct a vector query
+## 向量结构化匹配
 
-The following example filters the vectors with certain `book_id` values, and returns the `book_id` field and `book_intro` of the results.
+下面的示例过滤具有特定 `book_id` 的向量，并返回结果的 `book_id` field 和 `book_intro` field。
 
-Milvus supports setting consistency level specifically for a search or query  (only on PyMilvus currently). The consistency level set in the search or query requests overwrites the one set while creating the collection. In this example, the consistency level of the search request is set as "Strong", meaning Milvus will read the most updated data view at the exact time point when a search or query request comes. Without specifying the consistency level during a search or query, Milvus adopts the original consistency level of the collection.
+Milvus 支持专门为搜索或结构化匹配设置一致性级别（目前仅在 PyMilvus 上）。在创建 collection 时，搜索或结构化匹配请求中设置的一致性级别将覆盖设置的一致性级别。在本例中，搜索请求的一致性级别设置为 "Strong"，这意味着 Milvus 将在搜索或结构化匹配请求出现的确切时间点读取最新的数据视图。Milvus 在搜索或结构化匹配期间不指定一致性级别，而是采用 collection 的原始一致性级别。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/query.md:62]: 如果在搜索或结构化匹配期间不指定一致性级别，Milvus 将采用创建 collection 的原始一致性级别。

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Hey, @arthuo , I am glad that you're back. Did you go for your vacation?

I've made a few comments to your contribution.

**Comment by arthuo** [comment]
> Hey, @arthuo , I am glad that you're back. Did you go for your vacation?
> 
> I've made a few comments to your contribution.

Hello, it's been a long time. Recently, I have focused on the research of vision transformer, which has benefited me a lot.I'm glad we can fight shoulder to shoulder again.

**Comment by LocoRichard** [comment]
> > Hey, @arthuo , I am glad that you're back. Did you go for your vacation?
> > I've made a few comments to your contribution.
> 
> Hello, it's been a long time. Recently, I have focused on the research of vision transformer, which has benefited me a lot.I'm glad we can fight shoulder to shoulder again.

That's really cool, mate. I wish I could learn some more about computer science myself, haha.

Merging the PR.

**Pull request merged**